docs/README.md

```diff
@@ -57,6 +57,24 @@ Users of the Fuel indexer project include dApp developers looking to write flexi
 - We use Docker to produce reproducible environments for users that may be concerned with installing components with large sets of dependencies (e.g. Postgres).
 - Docker can be downloaded [here](https://docs.docker.com/engine/install/).
 
+### `postgresSQL` and `SQLite`
+
+- We use PostgresSQL and SQLite to manage our database in the following quickstart.
```

Comment:
@bitzoic 
- So Postgres and SQLite are actually not required for the Quickstart (because we use Docker), which is why we forgot to add them to this "Users" section of the README.
  - But they _should_ be in the "Users" section of the README (see comment above ☝️) - so good catch here.

Reply:
Ah, I only now saw that this is a "For Users" section. As per my other comment, since it's specifically a quickstart and not required I'll remove this.